modules/viantOrtbBidAdapter.js

```diff
@@ -51,8 +51,10 @@ export const spec = {
   onBidWon: function (bid) {
     if (bid.burl) {
       utils.triggerPixel(bid.burl);
+      utils.replaceAuctionPrice(bid.burl, bid.price);
```

Comment:
@skapoorViant, I aplogize for thinking about this only after merging.

I don't think this does what you want. it should probably be

```
utils.triggerPixel(utils.replaceAuctionPrice(bid.burl, bid.originalCpm || bid.cpm))
```

you can do the same for `nurl`, but by the time this runs it will be a duplicate call.

Reply:
no worries, but in the bidresponse we are sending bid.price and not bid.originalCpm or bid.cpm, so would it be good if I just write it as 
`utils.triggerPixel(utils.replaceAuctionPrice(bid.burl, bid.price||bid.originalCpm || bid.cpm))`